Doc/library/calendar.rst

```diff
@@ -404,7 +417,7 @@ The :mod:`calendar` module exports the following data attributes:
 .. data:: day_abbr
 
    A sequence that represents the abbreviated days of the week in the current locale,
-   where Mon is day number 0.
+   where Mon is day number ``0``.
```

Comment:
Maybe change this to Monday (or leave the part out altogether, as the fact that Monday is 0 has been mentioned already a couple of times). `Mon` is the abbreviation in my locale, but maybe not for all systems.
```suggestion
   where Monday is day number ``0``.
```